token-service/src/main/java/org/p2p/token/service/api/response/TokenPriceQueryResponse.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.token.service.api.response
+
+import com.google.gson.annotations.SerializedName
+
+internal data class TokenServiceQueryResponse<T>(
+    @SerializedName("chain_id")
+    val tokenServiceChainResponse: TokenServiceNetworkResponse,
+    @SerializedName("data")
+    val tokenServiceItemsResponse: List<T>
```

Comment:
can be just `items`
we are already in TokenService class, no need for that prefix